### ISSUE TITLE
Custom scalar of having inline description is Unknown type with Shopify Admin API

### ISSUE THREAD

**Issue by Yamashou** [jimkyndemeyer/js-graphql-intellij-plugin#371]
I'm using a [Shopify Admin API](https://shopify.dev/docs/admin-api). I don't get correct schema from the intorospection. Some custom scalars are unknown type. These scalar type have double quote in inline description.  
I would like the following solution:
- delete inline description type. I guess we need only multi line type. So I want to fix [this code](https://github.com/jimkyndemeyer/js-graphql-intellij-plugin/blob/90cf5fca904ba7fbe629d0bc02ed1279affd44d1/src/main/com/intellij/lang/jsgraphql/ide/editor/GraphQLIntrospectionService.java#L257). 
- replace `”` to `\"`
- If there is setting to skip description in config, please tell me that.

schema
```graphql
...

scalar FormattedString

"A string containing HTML code. Example value: `"<p>Grey cotton knit sweater.</p>"`."
scalar HTML

"A JSON Object. Example value: `{ "key1": "Value 1", "key2": "Value 2", "key3": 3 }`"
scalar JSON

"A monetary value string. Example value: `"100.57"`."
scalar Money

...

```

**Comment by vepanimas** [comment]
@Yamashou hi! The issue with quotes has already been fixed (#359), but has not yet been released. There are still some critical issues that should be fixed in version 2.6.0 😞.

Why do you think that we need only multiline descriptions? Single line descriptions are valid from the specification standpoint.

**Comment by Yamashou** [comment]
@vepanimas Thank you for quick response. #359 is awesome!!! 

> Why do you think that we need only multiline descriptions? Single line descriptions are valid from the specification standpoint.

I have presented it as one of the few solutions to this problem. I guess #359 is best solution. But  this solution has good points. I think it is good that the formats are unified, but I think it is not preferable because it is out of specifications. So don't worry too much.
